### PR TITLE
release on 11-30-2021

### PR DESCRIPTION
**Pull request by AshishMotanamGurunadham**
### Summary

- Release on 11-30-2021

### Deployment Link

https://terra-graphs-deployed-pr-#.herokuapp.com/

Thank you for contributing to Terra.
@cerner/terra
@cerner/carbon

### CODE REVIEW

File: packages/terra-graphs-docs/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cerner/terra-graphs-docs",
-  "version": "0.0.0",
+  "version": "0.0.1",
```
Comment (benbcai) [inline packages/terra-graphs-docs/package.json:3]: Do we want to make this version 1.0.0?

Comment (AshishMotanamGurunadham) [reply packages/terra-graphs-docs/package.json:3]: Not so sure about this, it is the version that is added by Ryan initially as 0.0.0 so I left it as is and it got updated to 0.0.1? 
https://github.com/cerner/terra-graphs/pull/246/files#diff-d761c3fb8ba076a072500ff322d01b361058688b30dff1e2cd8fd703d36b6da2R3

Comment (benbcai) [reply packages/terra-graphs-docs/package.json:3]: If terra-graph-docs is ready to go, the initial official release should be 1.0.0.

File: packages/terra-graphs-docs/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
   "private": true,
```
Comment (benbcai) [inline packages/terra-graphs-docs/package.json:4]: The `private` field should be deleted in order to release it.

File: packages/terra-graphs-docs/package.json (hunk 1 above)
Comment (AshishMotanamGurunadham) [reply packages/terra-graphs-docs/package.json]: Fixed [here](https://github.com/cerner/terra-graphs/pull/274/commits/81caeac55d11d0c6e5a2d9d6a8a544a65aaa6c2f#diff-d761c3fb8ba076a072500ff322d01b361058688b30dff1e2cd8fd703d36b6da2R3).

File: packages/terra-graphs-docs/package.json (hunk 2 above)
Comment (AshishMotanamGurunadham) [reply packages/terra-graphs-docs/package.json]: Fixed [here](https://github.com/cerner/terra-graphs/pull/274/commits/81caeac55d11d0c6e5a2d9d6a8a544a65aaa6c2f#diff-d761c3fb8ba076a072500ff322d01b361058688b30dff1e2cd8fd703d36b6da2L4).

File: packages/terra-graphs-docs/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## 0.0.1 - (November 30, 2021)
```
Comment (sdadn) [inline packages/terra-graphs-docs/CHANGELOG.md:5]: this should also be `1.0.0`

File: packages/terra-graphs-docs/package.json [hunk 4]
```diff
@@ -40,6 +39,5 @@
     "react": "^16.8.5",
     "react-dom": "^16.8.5"
   },
-  "scripts": {
-  }
+  "scripts": {}
```
Comment (benbcai) [inline packages/terra-graphs-docs/package.json:42]: Should this package include the `compile`, `lint`, and `precompile` scripts similar to other [terra docs packages](https://github.com/cerner/terra-core/blob/%40cerner/terra-core-docs%401.12.0/packages/terra-core-docs/package.json#L33-L35) and make sure these script can run successfully?

**Review by benbcai — APPROVED** [review]

File: packages/terra-graphs-docs/CHANGELOG.md (hunk 3 above)
Comment (AshishMotanamGurunadham) [reply packages/terra-graphs-docs/CHANGELOG.md]: Fixed [here](https://github.com/cerner/terra-graphs/pull/274/files#diff-dc22343a9102f2186f374011efbd61a1f3ac9df233e1e9d9385065fd4e2284d1R5).

**Review by sdadn — APPROVED** [review]

**Pull request merged**